src/events/messageCreate.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
         );
         talkedRecently.delete(message.author.id);
       }, __config.credits.timeout);
-    } else {
+    }
+ else {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 1 space.](https://app.codacy.com/gh/ZynerOrg/xyter/pullRequest?prid=9008883)
```suggestion
		else {
```